doc/Command-Reference.md

```diff
@@ -6818,7 +6838,7 @@ This command is used to add or delete the SNMP user for SNMPv3.
 
 - Usage:
   ```
-  config snmp user (add | del) <user> <noAuthNoPriv|AuthNoPriv|Priv> <RO|RW> <MD5|SHA|MMAC-SHA-2> <auth_password> <DES|AES> <encrypt_password>
+  config snmp user add <user> <noAuthNoPriv|AuthNoPriv|Priv> <RO|RW> [[<MD5|SHA|MMAC-SHA-2> <auth_password>] [<DES|AES> <encrypt_password>]]
```

Comment:
Same as above regarding `<noAuthNoPriv|AuthNoPriv|Priv>`, `<RO|RW>`, `<MD5|SHA|MMAC-SHA-2>` and `<DES|AES>`

Reply:
updated.